### PR TITLE
[fix] remove unused spaces in text

### PR DESCRIPTION
**Pull request by whiteshirt0429**

### CODE REVIEW

**Review by robin1001 — CHANGES_REQUESTED** [review]
please remove changes which are not related to this bug

**Comment by whiteshirt0429** [comment]
> please remove changes which are not related to this bug

done

File: examples/aishell/s0/run.sh [hunk 1]
```diff
@@ -27,11 +27,10 @@ train_config=conf/train_conformer.yaml
 checkpoint=
 cmvn=true
 dir=exp/sp_spec_aug
-
 # use average_checkpoint will get better result
 average_checkpoint=true
 decode_checkpoint=$dir/final.pt
-average_num=5
+average_num=10
```
Comment (glynpu) [inline examples/aishell/s0/run.sh:33]: why 10? the larger the better? what about 15?

**Review by glynpu — APPROVED** [review]

File: examples/aishell/s0/run.sh [hunk 2]
```diff
@@ -47,6 +46,13 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     utils/perturb_data_dir_speed.sh 0.9 data/train data/train_sp0.9
     utils/perturb_data_dir_speed.sh 1.1 data/train data/train_sp1.1
     utils/combine_data.sh data/train_sp data/train data/train_sp0.9 data/train_sp1.1
+    for x in train_sp dev test; do
```
Comment (placebokkk) [inline examples/aishell/s0/run.sh:49]: add comments like "Remove the space in Mandarin text.‘

**Review by placebokkk — CHANGES_REQUESTED** [review]

File: examples/aishell/s0/run.sh (hunk 1 above)
Comment (whiteshirt0429) [reply examples/aishell/s0/run.sh:33]: 10 can help get a more stable results

**Review by robin1001 — APPROVED** [review]

**Review by placebokkk — APPROVED** [review]

**Pull request merged**